assets/css/_late_view.scss

```diff
@@ -4,10 +4,15 @@ $late-view-background-color: $color-bg-light;
   @include modal-content;
   background-color: $late-view-background-color;
   height: 100%;
+  width: 33.5rem;
   position: absolute;
   right: 0;
   top: 0;
   z-index: 1001;
+
+  @media screen and (max-width: 700px) {
```

Comment:
question: This is close to, but not the same as `$mobile-max-width`. I assume it's intentional to use a different value in this one place?

Reply:
Correct, it's intentional based on how narrow the table can get and therefore when you start to run into issues with it being partially-hidden under the left nav bar, so specific to the content of this particular component.